### PR TITLE
Remove use of labeled.json for stats and rename functions

### PR DESCRIPTION
**Pull request by J535D165**

### CODE REVIEW

**Review by terrymyc — COMMENTED** [review]
Thanks. The corresponding tests in `test_api.py` also need to be modified.

File: asreview/webapp/api.py [hunk 1]
```diff
@@ -1046,23 +1046,22 @@ def api_get_progress_history(project_id):
...
+def api_get_progress_recall(project_id):
     """Get cumulative number of inclusions by ASReview/at random"""
 
     try:
-        statistics = get_data_statistics(project_id)
-        labeled = read_label_history(project_id)
-        data = []
-        for [key, value] in labeled:
-            data.append(value)
+
```
Comment (PeterLombaers) [inline asreview/webapp/api.py:1054]: This needs `project_path = get_project_path(project_id)`.

**Review by PeterLombaers — COMMENTED** [review]
One small issue, but looks good for the rest.

File: asreview/webapp/api.py (hunk 1 above)
Comment (J535D165) [reply asreview/webapp/api.py]: Thanks

**Pull request merged**